core/domain/exp_domain_test.py

```diff
@@ -3792,12 +3936,24 @@ class HTMLMigrationUnitTests(test_utils.GenericTestBase):
       html: <p>Hello, </p>this <i>is </i>state2
     content_ids_to_audio_translations:
       content: {}
+      default_outcome: {}
+      feedback_1: {}
+      feedback_2: {}
+      hint_1: {}
+      hint_2: {}
```

Comment:
I'm a little unfamiliar with the content ID system, but when we move feedback and hints around (or delete feedback causing everything to shift up by 1) what happens to the ID?

If the ID is typically an arbitrary random string then it should be an arbitrary random string here too. Otherwise this gives an impression that the IDs represent indexes.

If they do represent indexes, are there any tests that ensure that deleting feedback etc. leads to the correct result?